default.nix

```diff
@@ -21,7 +21,7 @@ let
       tasty-ant-xml = haskell.lib.doJailbreak super.tasty-ant-xml;
       binary = haskell.lib.dontCheck self.binary_0_8_4_1;
 
-      codec-jvm = self.callPackage ./utils/nix/codec-jvm.nix { };
+      codec-jvm = haskell.lib.dontHaddock (self.callPackage ./utils/nix/codec-jvm.nix { });
```

Comment:
I fixed this so you can probably revert this. You'll have to bump codec-jvm again.